WeakAuras/WeakAuras.lua

```diff
@@ -193,6 +193,19 @@ function SlashCmdList.WEAKAURAS(input)
     Private.PrintHelp();
   elseif msg == "repair" then
     StaticPopup_Show("WEAKAURAS_CONFIRM_REPAIR", nil, nil, {reason = "user"})
+  elseif msg == "optin" then
+    local feature = args[1]
+    if not Private.Features:Exists(feature) then
+      print(L["Unknown feature '%s'"]:format(feature))
+    else
+      if Private.Features:Enabled(feature) then
+        Private.Features:Disable(feature)
```

Comment:
It feels wrong to me that `/wa optin` is a toggle instead of having `optin` & `optout`

Reply:
I changed it to `ff`, for 'flag feature'. I don't care one way or another, if ppl want to bikeshed that I'm happy to change it to whatever people prefer.

Reply:
I agree with buds, enabling and disabling should be separate, otherwise telling users how to enable a feature is too complicated.